app/models/experimentation/notifications_experiment.rb

```diff
@@ -12,10 +12,12 @@ class NotificationsExperiment < Experiment
         .distinct
     end
 
+    # The order of operations is important.
+    # See https://docs.google.com/document/d/1IMXu_ca9xKU8Xox_3v403ZdvNGQzczLWljy7LQ6RQ6A for more details.
     def self.conduct_daily(date)
       running.each { |experiment| experiment.enroll_patients(date) }
       monitoring.each { |experiment| experiment.monitor(date) }
-      notifying.each { |experiment| experiment.send_notifications(date) }
+      notifying.each { |experiment| experiment.schedule_notifications(date) }
```

Comment:
I think I had asked this in a different PR as well: do we need to handle any errors here?